src/components/onboarding/index.tsx

```diff
@@ -126,7 +127,7 @@ export const Onboarding: React.FC = () => {
         {
           onSuccess: (data) => {
             if (!skipBilling(data as Account)) {
-              setState(claimedGrowth ? "skipped" : "plan");
+              setState(account?.trialPeriodEndedAt ? "skipped" : "plan");
```

Comment:
again calling out `account` can be async, is this the explicit behavior we want here?

Reply:
This one is a good callout, I think account _will be_ null here. I see from above that data is supposed to be account so I'll use `data?.trialPeriodEndedAt` instead and test it to see if this is the expected behaviour

Thanks again jonas for this one, good to know/be aware of 🙏